tokio-udp/src/recv.rs

```diff
@@ -0,0 +1,30 @@
+use super::UdpSocket;
+use std::future::Future;
+use std::io;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+/// A future that receives a datagram from the connected address.
+///
+/// This `struct` is created by [`recv`](super::UdpSocket::recv).
+#[must_use = "futures do nothing unless polled"]
+#[derive(Debug)]
+pub struct Recv<'socket, 'buf> {
```

Comment:
just to be consistent, Ive been just using `'a, 'b` instead of naming them.

Reply:
I used `<'socket, 'buf>` because that's what runtime uses. But `SendTo` have a third field `target`, and having `<'socket, 'buf, 'target>` seems a bit ugly, so I used `<'socket, 'b>` which is not consistent itself.

I think I will switch to just `<'a, 'b>`.